source/as-promise/types.ts

```diff
@@ -127,9 +127,9 @@ export interface PaginationOptions<ElementType, BodyType> {
 
 		/**
 		Defines how the parameter `allItems` in pagination.paginate, pagination.filter and pagination.shouldContinue is managed.
-		When set to `false`, the parameter `allItems` is always an empty array.
+		When set to `true`, the parameter `allItems` is an of the emitted items.
```

Comment:
Buttery butter. `allItems` is an array of all emitted items. hehe.

Reply:
Is this line change really needed?

Reply:
Fixed. And the parameters now are properties. @szmarczak @sindresorhus